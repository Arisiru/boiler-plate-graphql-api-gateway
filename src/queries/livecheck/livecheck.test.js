const resolver = require('./resolver');
const schema = require('./schema');

describe('[Query.livecheck]', () => {
  it('should compile schema', async () => {
    expect(schema).toMatchSnapshot();
  });

  it('should resolve field status', async () => {
    const result = await resolver.livecheck();

    expect(result.status).toEqual('OK');
  });
});
