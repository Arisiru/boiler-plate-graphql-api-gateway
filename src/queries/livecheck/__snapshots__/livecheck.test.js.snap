// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Query.livecheck] should compile schema 1`] = `
Object {
  "definitions": Array [
    Object {
      "directives": Array [],
      "fields": Array [
        Object {
          "arguments": Array [],
          "description": undefined,
          "directives": Array [],
          "kind": "FieldDefinition",
          "name": Object {
            "kind": "Name",
            "value": "livecheck",
          },
          "type": Object {
            "kind": "NonNullType",
            "type": Object {
              "kind": "NamedType",
              "name": Object {
                "kind": "Name",
                "value": "LivecheckResult",
              },
            },
          },
        },
      ],
      "interfaces": Array [],
      "kind": "ObjectTypeExtension",
      "name": Object {
        "kind": "Name",
        "value": "Query",
      },
    },
  ],
  "kind": "Document",
  "loc": Object {
    "end": 59,
    "start": 0,
  },
}
`;
